Formula/austin.rb

```diff
@@ -0,0 +1,24 @@
+class Austin < Formula
+  desc "Python frame stack sampler for CPython"
+  homepage "https://github.com/P403n1x87/austin"
+
+  url "https://github.com/P403n1x87/austin/archive/v1.0.1.tar.gz"
+  sha256 "f86804f777f2f466ddede5d530d3ca67582b2a1467d000662d81272d6e9c5639"
+
+  head "https://github.com/P403n1x87/austin.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "autoreconf", "--install"
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+    man1.install "debian/austin.1"
+  end
+
+  test do
+    shell_output("#{bin}/austin /usr/bin/python3 -c \"print('Test')\"", 33)
```

Comment:
I'm not sure if this test checks anything.
High Sierra and Mojave don't have `/usr/bin/python3` (and Catalina has it only if CLT is installed).

You may add `depends_on "python@3.8" => :test` and use real python installation `#{Formula["python@3.8"].opt_bin}/python3` in a test.

Reply:
Is there a reason why we can't check that `austin` really works? (and not exiting with 33 always).
We could write a simple python script and try to use `austin` for generating samples.

Reply:
Even with the dependency on Python 3.8, austin would require running the test with sudo. Whilst I could implement a simple test to check the output, as a user I'd be concerned if I had to use sudo. Otherwise, I'd be more than happy to improve testing. Also, note that the tarball is published to GitHub relesses by Travis upon a successful run of the full test suite